packages/commcare/src/Adaptor.js

```diff
@@ -77,23 +78,62 @@ export function get(path, params = {}, callback = s => s) {
       params
     );
 
+    let { offset = 0, limit = 1000 } = resolvedParams;
+
+    let nextState = state;
+    let result;
+    let allowPagination = !isNaN(resolvedParams.offset);
```

Comment:
I think this is backwards? If there's NO offset, we auto-paginate.

Unit tests should confirm this either way. Tests are passing so either the tests are wrong, or have insufficient coverage

Reply:
@josephjclark  If we change this then the tests will change as well. Let me make it that if we do not have `offset`, then we autopaginate